zip.go

```diff
@@ -93,10 +93,11 @@ func zipFile(w *zip.Writer, source string) error {
 
 		if baseDir != "" {
 			name, err := filepath.Rel(source, fpath)
+			name = strings.Replace(name, string(os.PathSeparator), "/", -1)
 			if err != nil {
```

Comment:
@Songmu filepath.Rel return a path string contains os.PathSeparator so it should be replace by "/"